### ISSUE TITLE
The insensitive trait just doesn't work

### ISSUE THREAD

**Issue by Manuelmc71** [ZioPao/The-Only-Cure#56]
there isn't much to it.
the trait dosn't work.

**Comment by ZioPao** [comment]
Hi, Insensitive was mostly used in the old version to combat Phantom Pain, but as of now I've scrapped the whole phantom pain system and it doesn't have a lot to do anymore. It's just a leftover at this point that I never reimplemented.

If you have any tips\considerations about it, I'm all ears

**Comment by Manuelmc71** [comment]
I personally like the trait itself and find it rather sad to remove it.
you should keep it like old but make it so that when you have this trait you don't scream when getting amputated, and make it so when you don't have it you scream

**Comment by ZioPao** [comment]
As of now, you don't actually scream when cutting off a limb. That function never actually worked afaik in the old version. 

In general, a trait to combat the pain would make sense, and in the future I may re-add\re-work the phantom pain system, but I was thinking of keeping it as just optional instead of forcing it.

**Comment by Manuelmc71** [comment]
yeah you could make it an option to turn on or off. (and if you do decide to add phantom pain again, you could add the mirror treatment to levitate to pain when first aid is high enough. tho it's just an idea.)
and for the screaming part, i know there are some mods out there (i don't remember which one exactly, perhaps schizophrenia) where the characters scream. you could check out their code.

**Comment by ScrooblesMcDoodle** [comment]
Hey, just chiming in on this discussion.
I noticed that the trait doesn't work either. It'd be nice if you removed it for the time being until you re-worked the pain system.
As a thought, you could potentially look into Arithmechick's Screams of Pain for reference, and make it so that cutting your arm off absolutely jacks up your pain meter straight to 100. When tied with the Screams of Pain mod, it would kick in and make your character scream aloud due to the way that mod handles pain.
